content/en/Integrations/IntegrationBuilder/How to Guides/github.md

```diff
@@ -0,0 +1,95 @@
+---
+title: "GitHub"
```

Comment:
As a generalized feedback: I've found that it helps people figure out where to click if I put a red box around the button they should click in screenshots. I add this using Preview (built into mac) then use the annotate > rectangle tool